pandas/core/arrays/interval.py

```diff
@@ -268,6 +275,9 @@ def _simple_new(
             # If these share data, then setitem could corrupt our IA
             right = right.copy()
 
+        dtype = IntervalDtype(left.dtype, closed=closed)
+        result._dtype = dtype
+
```

Comment:
should _closed now be unnecessary? as it can directly ref the dtype

Reply:
you're right, will update